File: app/src/main/java/com/example/servicebookingandroid/Model/UserDto.java
package com.example.servicebookingandroid.Model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserDto {
    private Long userId;

    private String username;

    private String firstname;

    private String lastname;

    private String streetname;

    private String city;

    private String state;

    private Integer zipcode;

    private String phone;

    private String language;

    private String role;


}

File: app/src/main/java/com/example/servicebookingandroid/Model/UserDto.kt
package com.example.servicebookingandroid.Model

data class UserDto(var userId: Long, var username: String, var firstname: String,
                   var lastname: String, var streetname: String, var city: String,
                   var state: String, var zipcode: Int, var phone: String, var language: String, var role: String) {


}